cmd/root.go

```diff
@@ -52,9 +52,12 @@ var rootCmd = &cobra.Command{
 		skipDeps, _ := cmd.Flags().GetBool("skip-deps")
 		diffRun, _ := cmd.Flags().GetBool("diff-run")
 
+		askUserToConfirm, _ := cmd.Flags().GetBool("asking-for-confirmation")
+
 		kappEngine = exec.NewKappEngine(exec.Opts{
-			DiffRun:   diffRun,
-			ExtraArgs: execExtraArgs,
+			DiffRun:                        diffRun,
+			ExtraArgs:                      execExtraArgs,
+			AskUserToConfirmBeforeKappExec: askUserToConfirm,
```

Comment:
This is also not consistent

Reply:
Fixed in https://github.com/DataReply/korgi/pull/15/commits/6134baeef0124ccf7393306f50571ce4e5e3028c.
Please check the new PR https://github.com/DataReply/korgi/pull/15, because I am going to remove my commits from this PR, as it was not open by me.